### PR TITLE
adjust build scripts for building all-in-one image

### PR DESCRIPTION
**Pull request by johnlanni**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:

we can use the all-in-one image in erda's self CICD pipeline.

### CODE REVIEW

File: build/dockerfiles/Dockerfile-all [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+RUN mkdir -p "$GOPATH/src/github.com/erda-project/erda/"
+COPY . "$GOPATH/src/github.com/erda-project/erda/"
+WORKDIR "$GOPATH/src/github.com/erda-project/erda/"
+
+RUN make build-all
+
+ARG BASE_DOCKER_IMAGE
+FROM ${BASE_DOCKER_IMAGE}
+
+RUN apk add --no-cache jq
```
Comment (innerpeacez) [inline build/dockerfiles/Dockerfile-all:13]: why install jq?

Comment (johnlanni) [reply build/dockerfiles/Dockerfile-all:13]: let @recallsong explain it

Comment (Effet) [reply build/dockerfiles/Dockerfile-all:13]: there's a temporary solution:
aim to boost speed of api-test, we make api-test runs in same process as the pipeline.

so jq called by pipeline directly...

and we are proposing the final solution, will be fixed in the next iteration.

Comment (luoyuctl) [reply build/dockerfiles/Dockerfile-all:13]: @recallsong

Comment (innerpeacez) [reply build/dockerfiles/Dockerfile-all]: > there's a temporary solution:
> aim to boost speed of api-test, we make api-test runs in same process as the pipeline.
> 
> we use jq to achieve some function like value-extracting, assertion... so jq called by pipeline directly...
> 
> and we are proposing the final solution, will be fixed in the next iteration.

got it

**Review by recallsong — APPROVED** [review]

**Pull request merged**